### DISCUSSION TITLE
我的货呀！！！

### DISCUSSION THREAD

**Discussion by Prance36** [share121/inter-knot#4912] [委托]
我听说有一家店的邦布插件特别好用，于是便在网上下单了一些，结果天马物流把我的货丢空洞里了，快来个人帮帮我！！！事成之后，必有重谢（以后再也不用天马物流了）

**Comment by BJ-cmyk** [comment]
委托，接，货，到手，钱？

**Reply by Prance36** [reply]
800菲林和150000丁尼，先给15%定金，事成之后，付尾款，你看怎么样？

**Reply by BJ-cmyk** [reply]
豪！🤓👍

**Reply by bugmaibear** [reply]
逆天

**Comment by Wattloo** [comment]
没人？打个郊先

**Comment by BJ-cmyk** [comment]
你买的邦布插件怎么是斐济杯插件呐？！😡这活我不干了！货扔到零号空洞里给以骸用了🤓你自己再买一个吧

**Reply by Prance36** [reply]
唉，不是，哥们，你找错箱子了！！！

**Comment by JT9364** [comment]
这个委托我接了

**Reply by Prance36** [reply]
可算又来帮忙的了，上一个还给我找错了，气死我了

**Comment by shaomai-maker** [comment]
是否支持原石结算,我这边正在和悬木人进行一些跨国贸易~
